btrack/core.py

```diff
@@ -685,7 +693,24 @@ def to_napari(
         ndim = self.configuration.volume.ndim if ndim is None else ndim
 
         return utils.tracks_to_napari(
-            self.tracks,
-            ndim=ndim,
-            replace_nan=replace_nan,
+            self.tracks, ndim=ndim, replace_nan=replace_nan
         )
+
+    def candidate_graph_edges(self) -> List[btypes.PyGraphEdge]:
+        """Return the edges from the full candidate graph."""
+        num_edges = self._lib.num_edges(self._engine)
+        if num_edges < 1:
+            logger.warning(
+                "No edges were found in the candidate graph. "
+                "``config.store_candidate_graph`` is set to "
+                f"{self.configuration.store_candidate_graph}"
+            )
+        graph_edges = [
+            self._lib.get_graph_edge(self._engine, idx)
+            for idx in range(num_edges)
+        ]
+        return graph_edges
+
+
+if __name__ == "__main__":
+    pass
```

Comment:
Why add this?

Reply:
dang it, I think that was from the original branch. removed